cl/search/management/commands/pacer_bulk_fetch.py

```diff
@@ -0,0 +1,385 @@
+            help="The minimum wait in secs between PACER fetches to the same court.",
+        )
+        parser.add_argument(
+            "--min-page-count",
+            type=int,
+            help="Get docs with this number of pages or more",
+        )
+        parser.add_argument(
+            "--max-page-count",
+            type=int,
+            help="Get docs with this number of pages or less",
+        )
+        parser.add_argument(
+            "--username",
+            type=str,
+            help="Username to associate with the processing queues (defaults to 'recap')",
+        )
+        parser.add_argument(
+            "--queue-name",
+            type=str,
+            help="Celery queue name used for processing tasks",
+        )
+        parser.add_argument(
+            "--testing",
+            type=str,
+            help="Prevents creation of log file",
+        )
+        parser.add_argument(
+            "--stage",
+            type=str,
+            choices=["fetch", "process"],
+            default="fetch",
+            help="Stage of the command to run: fetch or process",
+        )
+
+    @staticmethod
+    def docs_to_process_cache_key():
+        """Helper method to improve testability."""
+        return "pacer_bulk_fetch.docs_to_process"
+
+    @staticmethod
+    def timed_out_docs_cache_key():
+        """Helper method to improve testability."""
+        return "pacer_bulk_fetch.timed_out_docs"
+
+    @staticmethod
+    def setup_logging(testing: bool = False) -> None:
+        if not testing:
+            logging.basicConfig(
+                filename=f'pacer_bulk_fetch_{datetime.now().strftime("%Y%m%d_%H%M%S")}.log',
+                level=logging.INFO,
+                format="%(asctime)s - %(levelname)s - %(message)s",
+            )
+
+    def setup_celery(self) -> None:
+        """Setup Celery by setting the queue_name and throttle."""
+        self.queue_name = self.options.get("queue_name", "pacer_bulk_fetch")
+        self.throttle = CeleryThrottle(queue_name=self.queue_name)
+
+    def handle_pacer_session(self) -> None:
+        """Make sure we have an active PACER session for the user."""
+        self.pacer_username = self.options.get(
+            "pacer_username", settings.PACER_USERNAME
+        )
+        self.pacer_password = self.options.get(
+            "pacer_password", settings.PACER_PASSWORD
+        )
+        get_or_cache_pacer_cookies(
+            self.user.pk,
+            username=self.pacer_username,
+            password=self.pacer_password,
+        )
+
+    def set_user(self, username: str) -> None:
+        """Get user or raise CommandError"""
+        if not username:
+            raise CommandError(
+                "No username provided, cannot create PacerFetchQueues."
+            )
+        try:
+            self.user = User.objects.get(username=username)
+        except User.DoesNotExist:
+            raise CommandError(f"User {username} does not exist")
```

Comment:
I think you can just let it crash at this point, honestly, which makes this entire method unneeded.

Reply:
sure, method removed